packages/ngtools/webpack/src/transformers/find_image_domains.ts

```diff
@@ -0,0 +1,138 @@
+/**
+ * @license
+ * Copyright Google LLC All Rights Reserved.
+ *
+ * Use of this source code is governed by an MIT-style license that can be
+ * found in the LICENSE file at https://angular.io/license
+ */
+
+import * as ts from 'typescript';
+
+const TARGET_TEXT = '@NgModule';
+const BUILTIN_LOADERS = new Set([
+  'provideCloudflareLoader',
+  'provideCloudinaryLoader',
+  'provideImageKitLoader',
+  'provideImgixLoader',
+]);
+const URL_REGEX = /(https?:\/\/[^/]*)\//g;
```

Comment:
This regex is sort of a placeholder. I'll be updating this with a better one shortly, but I didn't want that to hold up discussion of the rest of the changes.